<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElfStructureView</class>
 <widget class="QWidget" name="ElfStructureView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="elfStructureSearchLine">
         <property name="placeholderText">
          <string>Search</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="elfStructureView">
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="animated">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTextBrowser" name="elfDetailView">
      <property name="openLinks">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionBack">
   <property name="icon">
    <iconset theme="go-previous"/>
   </property>
   <property name="text">
    <string>&amp;Back</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset theme="go-next"/>
   </property>
   <property name="text">
    <string>&amp;Forward</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
